src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -531,15 +544,24 @@ bool AdaptDispatch::_InsertDeleteHelper(const size_t count, const bool isInsert)
 {
     // We'll be doing short math on the distance since all console APIs use shorts. So check that we can successfully convert the uint into a short first.
     SHORT distance;
-    RETURN_BOOL_IF_FALSE(SUCCEEDED(SizeTToShort(count, &distance)));
+    if (FAILED(SizeTToShort(count, &distance)))
+    {
+        return false;
+    }
 
     // get current cursor, attributes
     CONSOLE_SCREEN_BUFFER_INFOEX csbiex = { 0 };
     csbiex.cbSize = sizeof(CONSOLE_SCREEN_BUFFER_INFOEX);
     // Make sure to reset the viewport (with MoveToBottom )to where it was
     //      before the user scrolled the console output
-    RETURN_BOOL_IF_FALSE(_pConApi->MoveToBottom());
-    RETURN_BOOL_IF_FALSE(_pConApi->GetConsoleScreenBufferInfoEx(csbiex));
+    if (!_pConApi->MoveToBottom())
```

Comment:
nit: bah, we don't have a RETURN_IF_FALSE? I rather wish we did, just so as to not explode all these lines out to four lines

Reply:
No, not really. I guess I could make one.